tests/integration/states/test_file.py

```diff
@@ -723,6 +723,29 @@ def test_managed_source_hash_indifferent_case(self):
             if os.path.exists(name):
                 os.remove(name)
 
+    @with_tempfile
```

Comment:
@rallytime this will need to become `@with_tempfile()` (i.e. with parenthesis) once merged into 2018.3.